resources/sass/pages/_cart.scss

```diff
@@ -0,0 +1,212 @@
+.flex {
+    @include flex;
+}
+
+.flex-row {
+    @include flex-row;
+}
+
+.flex-column {
+    @include flex-column;
+}
+
+.flex-row-reverse {
+    @include flex;
+    flex: 1;
+    flex-direction: row-reverse;
+}
+
+.flex-column-reverse {
+    @include flex;
+    flex: 1;
+    flex-direction: column-reverse;
+}
+
+main.cart {
+    @extend .flex-column;
+    background: #eeeeee;
+    padding: 33px 0px;
+    max-width: 1140px;
+    margin: 0 auto;
+    position: relative;
+    .cart-header {
+        @extend .flex-column;
```

Comment:
Stop this, it just cases duplication and this @extend way shouldn't be used for putting properties